core-library/src/main/java/org/silverpeas/core/index/search/model/QueryDescription.java

```diff
@@ -168,7 +167,7 @@ public void setRequestedLanguage(String requestedLang) {
    */
   public String getRequestedLanguage() {
     if (requestedLang == null) {
-      return Locale.getDefault().getLanguage();
+      return "*";
```

Comment:
This new default return value can cause some tricks. Indeed, this is not taken into account by `WAAnalyser` and in this case the stemming is by default in French.

Furthermore, I think the method `IndexSearcher#getPlainTextQuery` requires to be  touched so that the `Analyzer` used should match each language taken into account by this wildcard value ("*"); id est an `Analyzer` per language. In fact, in my advice, the `IndexSearcher` uses in a wrong way the `Analyzer` because a single instance of this analyzer is used for any language although an `Analyzer` instance is for a given language.

Reply:
OK.
I am not confortable with that seems to be requested, so I suggest you close this PR and you propose yours.

Reply:
After analysis and discussion with Miguel, I was finally able to propose a new solution.